app/models/form526_submission.rb

```diff
@@ -49,72 +49,15 @@ class Form526Submission < ApplicationRecord
   BIRLS_KEY = 'va_eauth_birlsfilenumber'
   SUBMIT_FORM_526_JOB_CLASSES = %w[SubmitForm526AllClaim SubmitForm526].freeze
 
+  # Called when the DisabilityCompensation form controller is ready to hand off to the backend
+  # submission process. Currently this passes directly to the retryable EVSS workflow, but if any
+  # one-time setup or workflow redirection (e.g. for Claims Fast-Tracking) needs to happen, it should
+  # go here and call start_evss_submission_job when done.
   def start
```

Comment:
This is where `SidekiqJobSelector` (now defunct and removed by this PR) redirects the default EVSS submission flow to the RRD flow, and sets up success/error handlers to rejoin the EVSS submission flow. With this code removed, there's no distinction between `#start` and `#start_evss_submission_job`. I opted to add an explanatory comment for now, instead of doing more refactoring.